i18n/locales/en.json

```diff
@@ -118,8 +118,8 @@
       "docs": "Docs"
     },
     "pagination": {
-      "next": "Newer | ",
-      "previous": "Older"
+      "next": "Next",
+      "previous": "Previous"
```

Comment:
`common.pagination` covered these same keys on lines 136 thru 142

Reply:
Right but this one is the legacy Next/Previous which is going to be removed once we remove the old website codebase.

Reply:
Then why change it now? 😁

Reply:
Please look carefully at the changes. These strings are used on the legacy blog pagination system.

And as you can see this PR overhauled the pagination system, meaning on the legacy blog the next and previous got switched positions, hence I had to remove the "|" character that was on the en.json file (tbh these sort of separators shouldnt be in a lang file)